official/resnet/imagenet_main.py

```diff
@@ -44,6 +44,19 @@
 
 DATASET_NAME = 'ImageNet'
 
+flags.DEFINE_bool('enable_lars',
```

Comment:
Can you move these into `define_resnet_flags` in `resnet_run_loop.py`?

Reply:
done